libs/base/src/lib/context/AccountProvider.tsx

```diff
@@ -257,19 +262,20 @@ const OnboardProvider: FC<{
       })
     }
 
-    fetch(`${API_ENDPOINT}/signature`)
-      .then(resp => resp.json())
-      .then(json => {
-        setStakeSignatures(prevSignatures => ({
-          ...prevSignatures,
-          message: json.message,
-        }))
-      })
-      .catch(console.error)
+    isGovernance &&
```

Comment:
```suggestion
    if (isGovernance) {
```
Might be nicer as a block here to improve readability & follow the pattern in the func